azavea/fb-gender-survey-dashboard
# Update Data

## Overview

The question text didn't appear for certain questions, showing only the
answer. This was the result of an error in the formatting of the
sog_agg_region.xlsx file/data.

Specifically, items c2_clean and c2_cook
had the questions/'Variable Label', "C.2 In normal circumstances, do
you take care of any of the following household chores at least half
of the time?" and "C.2 In normal circumstances, do you take care of
any of the following household chores at least half of the time?"
Each question code must have a period after the code in order for the
parsing we do to work, so they needed to read, "C.2. In normal
circumstances, do you take care of any of the following household
chores at least half of the time?" and "C.2. In normal circumstances,
do you take care of any of the following household chores at least half
of the time?"

The data has now been updated to correct the formatting, allowing these
questions to be parsed correctly.

~~Connects #XXX~~

### Demo

<img width="891" alt="Screen Shot 2021-01-27 at 3 49 19 PM" src="https://user-images.githubusercontent.com/21046714/106053135-70580e00-60b8-11eb-9b04-06b5eee97152.png">

<img width="882" alt="Screen Shot 2021-01-27 at 3 57 37 PM" src="https://user-images.githubusercontent.com/21046714/106053140-7221d180-60b8-11eb-96f6-51d4bcca4c2a.png">

## Testing Instructions

 * On staging, select a region. In the questions page, under 'Unpaid Caregiving', you will see several questions that contain no question text, and only answers. 
 * In the Netlify preview for this branch, select a region. Under 'Unpaid Caregiving', the questions should be displayed. 
 * Otherwise, the website should behave as before.
